com.unity.cinemachine/Runtime/PostProcessing/CinemachinePostProcessing.cs

```diff
@@ -258,7 +258,7 @@ static void ApplyPostFX(CinemachineBrain brain)
                     v.weight = b.m_Weight;
                     ++numPPblendables;
                 }
-#if true // set this to true to force first weight to 1
+#if false // set this to true to force first weight to 1
```

Comment:
Why not removing the code completely? This seems prone to mistakes.

Reply:
I'd rather leave it in for documentation purposes.  It's there to support a weirdness in PostProcessing, specifically for the DoF effect.  Clients are not expected to change this, unless they embed and customize CM - and then they're on their own.